### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by Sphakunene**
I am done. can you please review my code

### CODE REVIEW

File: frontend/ScanUI/components/User/LogIn.js [hunk 1]
```diff
@@ -1,11 +1,100 @@
...
+      <View style={styles.controlls}>
+        <TextInput placeholder="Enter username" onChangeText={usernameHandler} style={styles.textInput} />
+        <TextInput secureTextEntry={true} placeholder="Enter Password" onChangeText={passHandler}  style={styles.textInput} />
+        <TouchableOpacity onPress={()=>loginHandler()} style={styles.button}>
+          <Text style={styles.buttonText}>Login</Text>
+        </TouchableOpacity>
+        <Text style={{ fontSize: 18 }}>New to TapIT?</Text>
+        <TouchableOpacity>
+          <Text style={{ fontSize: 18, fontWeight: 'Bold', color: 'blue', textDecorationLine: 'underline', marginBottom: 20 }} >Create Account</Text>
+        </TouchableOpacity>
```
Comment (sallie325) [inline frontend/ScanUI/components/User/LogIn.js:36]: You are using inline styling when the styles should be applied externally

Comment (sallie325) [inline frontend/ScanUI/components/User/LogIn.js:36]: Missing onPress for navigating to the Create Profile page

**Review by sallie325 — CHANGES_REQUESTED** [review]

**Comment by Tshimangadzo** [comment]
Hi @Sphakunene 

Please checkout @sallie325 requested change and reopen this pull request after the changes.

File: frontend/ScanUI/components/User/LogIn.js (hunk 1 above)
Comment (Sphakunene) [reply frontend/ScanUI/components/User/LogIn.js:36]: I was using inline styling in those components that are of the type but they are style different .should I change the inline style to external?

**Comment by Sphakunene** [comment]
I have made requested changes

**Pull request merged**